.github/workflows/omega-docs-workflow.yml

```diff
@@ -62,16 +54,10 @@ jobs:
           git clone https://github.com/E3SM-Project/Omega.git --branch gh-pages --single-branch gh-pages
```

Comment:
This will need to change to `https://github.com/E3SM-Project/E3SM.git` when we merge into E3SM.